plugins/inputs/diskio/diskio_linux.go

```diff
@@ -15,7 +16,28 @@ type diskInfoCache struct {
 	values       map[string]string
 }
 
-var udevPath = "/run/udev/data"
+var udevPath = []string{"/run/udev/data", "/dev/.udev/db"}
+
+func getUdevDataPath(devName string, major uint64, minor uint64) (out string, err error) {
```

Comment:
So there are 4 paths can will be checked, for example:
- /run/udev/data/b8:0
- /run/udev/data/block:vda
- /dev/.udev/db/b8:0
- /dev/.udev/db/block:vda

Is this correct, I thought there was only these 2 that needed checked:
- /run/udev/data/b8:0
- /dev/.udev/db/block:vda

Reply:
You are correct that in a worst case scenario there are 4 path's that are being checked, this is because we cannot link the `b8:0` or `block:vda` to the udevPath[0] or udevPath[1] entry.

I experimented with hardcoding the paths and device format but this breaks the test scenarios.